k-distribution/tests/regression-new/set-symbolic-tests/failing/union-18-spec.k

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022 K Team. All Rights Reserved.
+// generated by gen-tests.sh
+
+requires "set-tests.k"
+
+module UNION-18-SPEC
+    imports SET-TESTS
+
+    claim <k> union ( S                    |Set (S  SetItem(X:MyId)) ) => . </k>
```

Comment:
There is an explicit simplification to rewrite `S |Set (S {X})` to `S {X}` but it somehow isn't applied.

Reply:
This is related to the bug https://github.com/runtimeverification/haskell-backend/issues/3301, rule was removed.